docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md

```diff
@@ -220,35 +220,38 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
 
 ![Screenshot Azure Arc-enabled server on resource group](./22.png)
 
-- You can also run _kubectl get nodes -o wide_ to check the cluster node status and _kubectl get pod -A_ to see that the cluster is running and all the needed pods (system, [the Arc-enabled Kubernetes extension pods](https://learn.microsoft.com/azure/azure-arc/kubernetes/extensions), and [Azure Monitor extension pods](https://learn.microsoft.com/azure/azure-monitor/containers/container-insights-overview) are in a running state.
+- You can also run _kubectl get nodes -o wide_ to check the cluster node status and _kubectl get pod -A_ to see that the cluster is running and all the needed pods (system and [the Arc-enabled Kubernetes extension pods](https://learn.microsoft.com/azure/azure-arc/kubernetes/extensions)) are in a running state.
 
 ![Screenshot kubectl get nodes -o wide](./36.png)
```

Comment:
screenshot numbers are jumping around and not sequential. Can you please fix the numbers so they are aligned? I know its an annoying thing but it can be confusing if people are just viewing the screenshot files outside of the readme.